typescript/sdk/src/consts/chains.ts

```diff
@@ -59,12 +59,12 @@ export const Testnets = [
   Chains.moonbasealpha,
   Chains.mumbai,
   Chains.optimismgoerli,
-] as Array<ChainName>;
+] as Array<CoreChainName>;
 
 export const TestChains = [
   Chains.test1,
   Chains.test2,
   Chains.test3,
-] as Array<ChainName>;
+] as Array<CoreChainName>;
```

Comment:
imo this typing is misleading

Reply:
As in you'd prefer it rename ChainName? CoreChainName is maybe misleading but more correct IMO, in the sense that a 'core' chain is just one the SDK knows about. I can rename to `SdkChainName` if you prefer?

Reply:
Resolved offline